mrtt-ui/src/components/SiteBackgroundForm.js

```diff
@@ -79,7 +79,7 @@ const ProjectDetailsForm = () => {
     if (!data) return
 
     axios
-      .put(url, mapDataForApi('siteBackground', data))
+      .patch(url, mapDataForApi('siteBackground', data))
       .then(() => {
```

Comment:
put will overwrite the rest of the registration answers with nothing. PAtch inserts and leaves other answers alone.